frontend/src/main/components/Jobs/ClearJobLogsForm.js

```diff
@@ -0,0 +1,28 @@
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+
+const ClearJobLogsForm = ({ callback }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    callback();
+  };
+  // Stryker disable all
```

Comment:
Why do we have a // Stryker disable all here?

This isn't one of the cases that we've approved as an exception.

Please remove this and try to write proper tests so that this code is covered.

Reply:
I believe I was told that Stryker here is testing different styles of padding but since this is a visual optimization it is okay. Is this not the case?

Reply:
<img width="589" alt="image" src="https://github.com/user-attachments/assets/726c55ed-ddea-43b1-9de2-66d04b519828">
Getting this when I don't disable stryker and I'm not sure if this is intended or not